src/include/daos_errno.h

```diff
@@ -245,7 +245,10 @@ extern "C" {
 	       Agent is incompatible with libdaos)			\
 	/** Multiple shards locate on the same target */		\
 	ACTION(DER_SHARDS_OVERLAP,	(DER_ERR_DAOS_BASE + 30),	\
-	       Shards overlap)
+	       Shards overlap)						\
+	/** Not sure whether the TX is committable or not */		\
+	ACTION(DER_TX_UNCERTAINTY,	(DER_ERR_DAOS_BASE + 31),	\
```

Comment:
The term "uncertainty" is already used for checking time skew in distributed transaction, I think we'd use some different term here to avoid confusion.

Reply:
To avoid confusion, we add the prefix "_TX_", just as "DER_TX_RESTART" does. I am not against to use another term, but not sure which one is better. Do you have any suggested name?

Reply:
DER_TX_UNKNOWN ?  I agree that uncertain is confusing since we use DER_TX_RESTART if we hit an uncertainty violation